rocketpy/Environment.py

```diff
@@ -378,14 +385,17 @@ def __init__(
 
         return None
 
-    def setDate(self, date):
+    def setDate(self, date, time_zone=None):
```

Comment:
Can we make it so that the default entry for the time_zone argument is "UTC" instead of `None`? If so, I believe we wouldn't need the `if` statement in line 407, making the code cleaner. (I don't know if it would work as expected...)